packages/loot-core/src/types/models/reports.d.ts

```diff
@@ -38,6 +38,7 @@ export interface GroupedEntity {
 
 type LegendEntity = {
   name: string;
+  id: string;
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
  id: string | null;
```